test/modules/fundingManager/RebasingFundingManager.t.sol

```diff
@@ -46,7 +46,7 @@ contract RebasingFundingManagerTest is ModuleTest {
     uint internal constant DEPOSIT_CAP = 100_000_000e18;
 
     // Other constants.
-    uint8 private constant DECIMALS = 18;
+    //uint8 private constant DECIMALS = 18;  //Question : Is there any reason to hardcode the decimals for this fundingmanager to 18?
```

Comment:
Is this for the token that is being deposited or the receipt token? If it's for the deposited token, then it shouldn't be hardcoded, right?

Reply:
It's for the token that the bonding curve will issue. But I guess since we are going to make changes anyway for minting from an existing token, we shouldn't hardcode it